Makefile

```diff
@@ -80,10 +80,6 @@ ifeq ($(TARGET_ARCH),arm64)
 endif
 endif
 
-ifeq ($(TARGET_OS),linux)
```

Comment:
needed to add these to fix CI now the our github workflows are going from using ubuntu 22.04.x to 24.04.x